azure-storage-cpp-lite/src/blob/blob_client_wrapper.cpp

```diff
@@ -10,8 +13,10 @@
 
 namespace microsoft_azure {
     namespace storage {
+
+const unsigned long long DOWNLOAD_CHUNK_SIZE = 16 * 1024 * 1024;
```

Comment:
spacing

Reply:
not sure what spacing to use -) it's not consistent though this file. would you object auto reformating the file?

Reply:
Space instead of tabs is what we're aiming to use. Yes to auto reformat! :)

Reply:
sure, will reformat the file before submitting a merge

Reply:
as promised, reformatted the file